app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -298,5 +325,161 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
     }
   };
 
+  $scope.populatePostValidationModel = function() {
+    if ($scope.emsCommonModel.default_auth_status === true) {
+      $scope.postValidationModelRegistry("default");
+    }
+    if ($scope.emsCommonModel.amqp_auth_status === true) {
+      $scope.postValidationModelRegistry("amqp");
+    }
+    if ($scope.emsCommonModel.service_account_auth_status === true) {
+      $scope.postValidationModelRegistry("service_account");
+    }
+    if ($scope.emsCommonModel.metrics_auth_status === true) {
+      $scope.postValidationModelRegistry("metrics");
+    }
+    if ($scope.emsCommonModel.ssh_keypair_auth_status === true) {
+      $scope.postValidationModelRegistry("ssh_keypair");
+    }
+    if ($scope.emsCommonModel.hawkular_auth_status === true) {
+      $scope.postValidationModelRegistry("hawkular");
+    }
+  };
+
+  $scope.postValidationModelRegistry = function(prefix) {
+    if (!angular.isDefined($scope.postValidationModel)) {
+      $scope.postValidationModel = {default: {},
+                                    amqp: {},
+                                    metrics: {},
+                                    ssh_keypair: {},
+                                    hawkular: {}}
+    }
+    if (prefix === "default") {
+      if ($scope.newRecord) {
+        var default_password = $scope.emsCommonModel.default_password;
+        var default_verify = $scope.emsCommonModel.default_verify;
+      } else {
+        var default_password = $scope.emsCommonModel.default_password === "" ? "" : miqService.storedPasswordPlaceholder;
+        var default_verify = $scope.emsCommonModel.default_verify === "" ? "" : miqService.storedPasswordPlaceholder;
```

Comment:
First off, note that the backend *never* sends the password -- the only info that the frontend has is that password exists for a particular credential record, and that's it!
So, as you can imagine, the point of not showing the actual password and showing the placeholder instead is two-fold -
(a) Security
(b) illusion of a password being present in that field
Also note that the placeholder represents ng-model for the password field. 

The `console.log` above will just return you the placeholder value